main/politeness_logi.R

```diff
@@ -0,0 +1,43 @@
+library(pscl)
+library(plyr)
+library(readr)
+
+# get data
+dat <- read.csv("/dev/stdin")
+dim(dat)
+# collect counts of marked politness words and store them in a file
+system2("src/convo_politeness",
+        stdout = "politeness_features.csv",
+        input = format_csv(dat))
+
+# redirect output to a file
+sink("politeness_logi.out")
```

Comment:
should this maybe be before the system2 call? one might worry some standard output starts before the sink call begins

Reply:
I've moved it up.